README.markdown

```diff
@@ -117,7 +114,54 @@ Storm/Mesos provides resource isolation between topologies. So you don't need to
 * `topology.mesos.worker.mem.mb`: Memory (in MiB) per worker. Defaults to "1000".
   * `worker.childopts`: Use this for JVM opts.  You should have about 25% memory overhead for each task.  For
   example, with `-Xmx1000m`, you should set `topology.mesos.worker.mem.mb: 1250`
-* `topology.mesos.executor.cpu`: CPUs per executor. Defaults to "1".
-* `topology.mesos.executor.mem.mb`: Memory (in MiB) per executor. Defaults to "1000".
+* `topology.mesos.executor.cpu`: CPUs per executor. Defaults to "0.1".
```

Comment:
Do we really need to change this to 0.1? This would break the contract for all the existing users. By users, I mean those who are running topologies on mesos/storm cluster.